app/views/barcode_items/index.html.erb

```diff
@@ -5,7 +5,13 @@
   <h2>Filters</h2>
   <%= form_tag(barcode_items_path, method: :get) do |f| %>
     <%= label_tag "By Item Category" %>
-    <%= collection_select(:filters, :item_id, @items, :id, :name) %>
+    <%= collection_select(:filters, :item_id, @items, :id, :name, include_blank: true) %>
+
+    <%= label_tag do %>
```

Comment:
Ah, I understand now why the @global instance variable in the index action.

I think as our list of global bar codes fills up we may want to change it to only show local as diaper banks will probably start taking it for granted that any global bar code is going to exist.

Reply:
I've got some ideas for improving the `global` issue -- ie. constraining the behavior of it and allowing a "submit to be considered for global" kinda thing, but we can iterate that at a later date :)